web/source/kmwnative.ts

```diff
@@ -235,7 +236,10 @@ namespace com.keyman.osk {
       w1 = w2 = wx;    
       ctx.fillStyle = '#999';
     } else {
-      ctx.fillStyle = '#ffffff';
+      // #0f1319 is the current dark-mode background color set in kmwosk.css.
+      // There might be a way to automatically retrieve it, but that'll take a
+      // bit of research to find.
+      ctx.fillStyle = util.prefersDarkMode() ? '#0f1319' : '#ffffff';
```

Comment:
I think it would be good to be factoring these types of constants out of the code here and either to the top of the file or into a separate file. I don't have a huge problem with the disconnect between this and the CSS; it's not ideal, of course, but the work to unify this is significant and not really worthwhile.

Reply:
:+1:  See #2484.